common/src/main/java/com/github/dominik48n/party/user/UserDeserializer.java

```diff
@@ -19,19 +19,22 @@ public PartyPlayer deserialize(final JsonParser parser, final DeserializationCon
         final UUID uniqueId = UUID.fromString(node.get("uuid").asText());
         final String name = node.get("name").asText();
         final UUID partyId = node.hasNonNull("party_id") ? UUID.fromString(node.get("party_id").asText()) : null;
-        return new DeserializedUser(uniqueId, name, partyId);
+        final int memberLimit = node.get("member_limit").asInt();
+        return new DeserializedUser(uniqueId, name, partyId, memberLimit);
     }
 
-    private static class DeserializedUser implements PartyPlayer {
+    static class DeserializedUser implements PartyPlayer {
 
         private final @NotNull UUID uniqueId;
         private final @NotNull String name;
+        private final int memberLimit;
         private @Nullable UUID partyId;
 
-        public DeserializedUser(final @NotNull UUID uniqueId, final @NotNull String name, final @Nullable UUID partyId) {
+        public DeserializedUser(final @NotNull UUID uniqueId, final @NotNull String name, final @Nullable UUID partyId, final int memberLimit) {
```

Comment:
Auch hier wäre meine Präferenz ein Konstruktor generiert durch Lombok

Reply:
The reason why this is not implemented can be found in the comment in the pull request.